README.md

```diff
@@ -89,6 +89,20 @@ This config option controls the export file format for each Sphinx builder. It
 accepts a dictionary mapping builder names to image formats. Accepted values for
 the latter are `"png"`, `"jpg"`, or `"svg"`.
 
+### Output directory
+- *Formal Name*: `drawio_export_directory`
+- *Default Value*: `None`
+
+This config option controls where the generated files are exported to.
+Both relative(relative to the current working directory) and absolute paths
+are supported.
+
+If not set all files are generated into the builders output directory
+e.g. `_build/html/.drawio/*`
+
+This option might be beneficial to save conversion time when using multiple
```

Comment:
```suggestion
This can be useful to reduce build times with many drawio files across many sphinx build formats.
```
Still feels a strange sentence to me,

Reply:
How about _"This can reduce disk space usage and build times as the exported images can be reused among Sphinx builders, provided the builders use the same drawio export format"_?

However, that probably still doesn't make it clear what exactly is happening behind the scenes and thus may be confusing. But then again, does the average user need to worry about that?